### PR TITLE
PB-6765: Added env QUERY_MONGO_DB to false for fresh installs

### PR DESCRIPTION
**Pull request by px-kesavan**
**What this PR does / why we need it**:  Added env QUERY_MONGO_DB to false for fresh installs

**Which issue(s) this PR fixes** (optional)
Closes # 

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml [hunk 1]
```diff
@@ -148,6 +148,10 @@ spec:
           - name: SSL_CERT_DIR
             value: /tmp/certs
           {{- end }}
+          {{- if eq .Release.IsInstall true }}
+          - name: QUERY_MONGO_DB
```
Comment (kravichandran-px) [inline charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml:152]: Is this only for  middleware use case? If yes, should we rename this to be more specific to that use case?

File: charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml [hunk 2]
```diff
@@ -148,6 +148,10 @@ spec:
...
+            value: "false"
```
Comment (kravichandran-px) [inline charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml:153]: when will we set this to true?

Comment (kravichandran-px) [inline charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml:153]: lets create a task to remove this for all the further releases. Else I guess a customer who installed 2.7.0 fresh, but upgrades to 2.7.1 will end-up querying mongodb, which is not needed.

**Review by kravichandran-px — CHANGES_REQUESTED** [review]

File: charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml (hunk 1 above)
Comment (px-kesavan) [reply charts/px-central/templates/px-lighthouse/px-central-ui/pxcentral-lighthouse.yaml:152]: Yes, it only for middleware . This ENV is part of the middleware codebase . So i am reluctant in changing it

**Review by t-nidhin — APPROVED** [review]

**Pull request merged**